sdk/log/internal/x/README.md

```diff
@@ -0,0 +1,34 @@
+# Experimental Features
+
+The log SDK contains features that have not yet stabilized.
+These features are added to the OpenTelemetry Go log SDK prior to stabilization so that users can start experimenting with them and provide feedback.
+
+These feature may change in backwards incompatible ways as feedback is applied.
+See the [Compatibility and Stability](#compatibility-and-stability) section for more information.
+
+## Features
+
+- [Filter Processors](#filter-processor)
+
+### Filter Processor
+
+Users of logging APIs often want to know if a log `Record` will be processed or dropped before they perform complex operations to construct the `Record`.
```

Comment:
"logging APIs" -> "logging libraries" per naming in https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/bridge-api.md and https://github.com/open-telemetry/opentelemetry-go/blob/main/log/doc.go

```suggestion
Users of logging libraries often want to know if a log `Record` will be processed or dropped before they perform complex operations to construct the `Record`.
```